lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -50,7 +50,17 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
-  <button class="button button-primary button-block"
+  <div class="modal-footer-warning"
+    ng-show="modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive())">
+      Etcher has identified this as a system drive
+  </div>
+  <button class="button button-block"
+    ng-class="{ 'label-danger': modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive()),
+                'button-primary': modal.state.hasDrive() && !modal.state.isSystemDrive(modal.state.getDrive()) }"
```

Comment:
Oh I see. In that case, if you create a function in the controller to determine if `danger` should be applies, then you can move that logic there (add an `if` conditional or something), and then simplify these expressions.